_includes/category_links.html

```diff
@@ -1,8 +0,0 @@
-{%- if categories.size > 0 -%}
-  <p class="categories-info">on
-    {% for category in categories %}
-      <a class="btn-link" href="{{ site.baseurl }}/tags/{{ category | downcase | replace: ' ','-' }}"
-         title="Articles on {{ category | titleize | replace: '-',' ' }}">{{ category | titleize | replace: '-',' ' }}</a>
-     {%- unless forloop.last -%},{%- endunless -%}{% endfor %}
-   	</p>
-{%- endif -%}
```

Comment:
@nykka Why this file was removed? I see that in `_includes/author.html` you are using this whole code twice